tensorboard/plugins/mesh_visualizer/tf_mesh_dashboard/tf-mesh-loader.html

```diff
@@ -0,0 +1,81 @@
+<!--
+@license
+Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<link rel="import" href="../paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-backend/tf-backend.html">
+<link rel="import" href="../tf-card-heading/tf-card-heading.html">
+<link rel="import" href="../tf-color-scale/tf-color-scale.html">
+<link rel="import" href="../tf-imports/lodash.html">
+<link rel="import" href="../tf-imports/threejs.html">
+<link rel="import" href="tf-mesh-viewer.html">
+
+<!--
+  Mesh component loads 3D data (point clouds, meshes, etc.) and renders it.
+-->
+<dom-module id="tf-mesh-loader">
+  <template>
+    <tf-card-heading
+      color="[[_runColor]]"
+      class="tf-mesh-loader-header"
+    >
+      <template is="dom-if" if="[[_hasMultipleSamples]]">
+        <div>sample: [[_sampleText]] of [[ofSamples]]</div>
+      </template>
+      <template is="dom-if" if="[[_computeHasAtLeastOneStep(_steps)]]">
```

Comment:
`compute` means something specific in context of Polymer. Would you mind changing it a bit?

Reply:
I made it by example of other plugins. What would you prefer to have instead? I can just remove _compute prefix and convert all properties to methods if this seems like a better design.